src/dbus/server/dbus_thread_object.cpp

```diff
@@ -1410,6 +1413,25 @@ otError DBusThreadObject::GetDnssdCountersHandler(DBusMessageIter &aIter)
 #endif // OTBR_ENABLE_DNSSD_DISCOVERY_PROXY
 }
 
+otError DBusThreadObject::GetTelemetryDataHandler(DBusMessageIter &aIter)
+{
+#if OTBR_ENABLE_FEATURE_FLAGS
```

Comment:
I wonder how this is related to feature flags?

Reply:
Done.

Reply:
Added OTBR_ENABLE_TELEMETRY_DATA_API flag.